engine/src/main/java/org/terasology/recording/CharacterStateEventPositionMap.java

```diff
@@ -22,8 +22,9 @@
 import java.util.Map;
 
 /**
- * Saves the "position" and "velocity" variable for every CharacterStateEvent so that the player location can be
- * reproduced correctly during a replay.
+ * When a CharacterMoveInputEvent is caught by ServerCharacterPredictionSystem, a new CharacterStateEvent is created and
```

Comment:
General note for javadoc: Try to come up with a once sentence summary  that fits on one line and ends with a `.` - that goes into the summary section for the class in the javadoc site. If you end up with a multi-sentence summary only the first sentence makes it which can make the docs read kinda funny.

Then get into as much detail as you like after the first sentence/paragraph 👍